File: fetch_images.js
const https = require('https');

function search(query) {
  return new Promise((resolve) => {
    https.get('https://html.duckduckgo.com/html/?q=' + encodeURIComponent(query + ' pinterest.com aesthetic wallpaper'), (res) => {
      let data = '';
      res.on('data', chunk => data += chunk);
      res.on('end', () => {
        const matches = data.match(/src="\/\/external-content\.duckduckgo\.com\/iu\/\?u=([^&"]+)/g);
        if (matches && matches.length > 0) {
          const rawUrl = decodeURIComponent(matches[0].split('=')[1]);
          console.log(query + ':', rawUrl);
        } else {
          console.log(query + ':', 'None');
        }
        resolve();
      });
    });
  });
}

async function run() {
  await search('messi world cup');
  await search('cristiano ronaldo real madrid');
  await search('neymar brazil');
  await search('paolo maldini ac milan');
  await search('football jersey rack aesthetic');
  await search('retro football jersey aesthetic');
  await search('football stadium cinematic aesthetic');
  await search('national football team aesthetic');
  await search('football player edition jersey aesthetic');
}
run();

File: fetch_images.py
import urllib.request, re

def get_images(query):
    url = f'https://wallpapercave.com/search?q={query.replace(" ", "+")}'
    req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
    try:
        html = urllib.request.urlopen(req).read().decode()
        matches = re.findall(r'src="(/wp/[^"]+\.jpg)"', html)
        return ['https://wallpapercave.com' + m for m in matches[:3]]
    except Exception as e: return []

print('Messi:', get_images('messi world cup aesthetic'))
print('Ronaldo:', get_images('cristiano ronaldo real madrid aesthetic'))
print('Neymar:', get_images('neymar aesthetic'))
print('Maldini:', get_images('paolo maldini aesthetic'))
print('Jersey Rack:', get_images('football jersey collection'))
print('Retro Jersey:', get_images('retro football shirt'))
print('Stadium:', get_images('stadium cinematic aesthetic'))
print('Hero:', get_images('football tunnel cinematic aesthetic'))
